lib/src/components/NxTransferList/TransferListHalf.tsx

```diff
@@ -27,11 +39,36 @@ function TransferListItem<T extends string | number = string>(props: TransferLis
 
   return (
     <NxOverflowTooltip>
-      <label className="nx-transfer-list__item">
-        <NxFontAwesomeIcon icon={checked ? faTimesCircle : faPlusCircle} />
-        <input className="nx-transfer-list__checkbox" type="checkbox" checked={checked} onChange={onChange} />
-        <span>{displayName}</span>
-      </label>
+      <div className={classnames('nx-transfer-list__item',
+          { 'nx-transfer-list__item--with-reordering': !!showReorderingButtons })}>
+        <label className="nx-transfer-list__select">
+          <NxFontAwesomeIcon icon={checked ? faTimesCircle : faPlusCircle} />
+          <input className="nx-transfer-list__checkbox" type="checkbox" checked={checked} onChange={onChange} />
+          <span>{displayName}</span>
+        </label>
+        { showReorderingButtons && (
+          <NxButtonBar className="nx-transfer-list__button-bar">
+            <NxButton variant="icon-only"
+                      className={classnames('nx-transfer-list__button',
+                          'nx-transfer-list__button--move-up', {
+                            'disabled': isTopItem
```

Comment:
I think we probably want to disable these via attr, not classname.  There's no reason for them to be focusable when disabled I don't think - the tooltips on these aren't helpful on the disabled ones.

Reply:
> I think we probably want to disable these via attr, not classname. There's no reason for them to be focusable when disabled I don't think - the tooltips on these aren't helpful on the disabled ones.

That's a good point: I updated it here: 8da7e29ba73263e6ce9f96e12d36ba9915305d0f